A32NX/html_ui/Pages/A32NX_Core/A32NX_TransitionAltitude.js

```diff
@@ -0,0 +1,345 @@
+class A32NX_TransitionAltitude {
+    init() {
+        console.log('A32NX_TransitionAltitude init');
+        this.currentDeparture = "";
+        this.currentArrival = "";
+        this.offline = false;
+        this.checkStart = true;
+        this.checkStartSec = 0;
+        this.testAirport = "";
+        this.trySendRequest();
+    }
+
+    update(_deltaTime, _core) {
+        if (this.offline === true) {
+            this.offlineTransAlt();
+        }
+        this.tryCheckAPI(_deltaTime);
+    }
+
+    async trySendRequest() {
+        await NXApi.getAirport("KLAX")
+            .then((data) => {
+                this.testAirport = data.transAlt;
+            });
+    }
+
+    tryCheckAPI(_deltaTime) {
+        if (this.offline === false && this.checkStart === true) {
+            this.checkStartSec += _deltaTime;
+        }
+        if ((this.checkStartSec >= 60 * 1000) && (this.testAirport === "")) {
+            this.offline = true;
+            this.checkStart = false;
+        } else if ((this.checkStartSec >= 60 * 1000) && (this.testAirport !== "")) {
+            this.offline = false;
+            this.checkStart = false;
+            this.checkStartSec = 0;
+        }
+    }
+
+    offlineTransAlt() {
+        const Departure = NXDataStore.get("PLAN_ORIGIN", "");
+        const Arrival = NXDataStore.get("PLAN_DESTINATION", "");
```

Comment:
**issue**: wrong variable name capitalisation.
**thought**: given that you get an origin and destination, it probably makes sense to use the same name for the variables.

Reply:
Ok

Reply:
You resolved this without resolving it in code.

Reply:
what? this is outdated. i change to origin & destination